rusk-recovery/src/state/snapshot/governance.rs

```diff
@@ -6,32 +6,38 @@
 
 use dusk_bls12_381_sign::PublicKey as BlsPublicKey;
 use dusk_bytes::Serializable;
-use dusk_pki::PublicKey;
-use rusk_abi::ModuleId;
+use dusk_pki::{PublicKey, PublicSpendKey};
+use rusk_abi::ContractId;
 use serde_derive::{Deserialize, Serialize};
 
 use super::wrapper::Wrapper;
 
 #[derive(Serialize, Deserialize, PartialEq, Eq)]
 pub struct Governance {
+    pub(crate) contract_owner:
+        Wrapper<PublicSpendKey, { PublicSpendKey::SIZE }>,
```

Comment:
Can you update the `example_gov.toml` to support this mandatory field?

Reply:
To be on the same page
You didn't make it always mandatory, it's mandatory only if you add any governance entry